FlexiblePopupPanels/Panel.js

```diff
@@ -153,7 +153,7 @@ const PanelBase = kind({
 
 	handlers: {
 		onNextClick: handle(
-			adaptEvent(() => ({type: 'onNextClick'}), forwardWithPrevent('onNextClick')),
+			forwardCustomWithPrevent('onNextClick'),
 			adaptEvent(
```

Comment:
Can we replace this with `forwardCustom`?

Reply:
> Thank you @MikyungKim! I will check listed ones. 👍

Listed cases are all fixed.

Reply:
✅